docker/dev.Dockerfile

```
@@ -0,0 +1,11 @@
+FROM ghcr.io/alphagov/datagovuk_publish:v2.16.0
+
+USER root
+ENV BUNDLE_WITHOUT=""
+ENV RAILS_ENV=development
+RUN apt update && install_packages \
```

[install_packages](https://github.com/alphagov/govuk-ruby-images/blob/main/install_packages.sh#L11) already takes care of the `apt update`.

```suggestion
RUN install_packages \
```

---

ok, I've taken it out